src/common/opportunity.js

```diff
@@ -71,7 +71,8 @@ export async function convertToOpportunity(auditUrl, auditData, context, createO
       opportunity.setAuditId(auditData.id);
       if (auditType === Audit.AUDIT_TYPES.CWV
           || auditType === Audit.AUDIT_TYPES.META_TAGS
-          || auditType === Audit.AUDIT_TYPES.SECURITY_CSP) {
+          || auditType === Audit.AUDIT_TYPES.SECURITY_CSP
+          || auditType === 'security-vulnerabilities') { // TODO: Audit.AUDIT_TYPES.SECURITY_VULNERABILITIES
```

Comment:
this needs a PR + release + dependency bump on https://github.com/adobe/spacecat-shared/blob/main/packages/spacecat-shared-data-access/src/models/audit/audit.model.js

Reply:
not sure if we can trigger it via Slackbot etc without adding it to the model - never tested

Reply:
PR opened https://github.com/adobe/spacecat-shared/pull/953